src/proto/message.proto

```diff
@@ -102,6 +102,8 @@ enum UserDataType {
   USER_DATA_TYPE_LOCATION = 7; // Current location for the user
   USER_DATA_TYPE_TWITTER = 8; // Username of user on twitter
   USER_DATA_TYPE_GITHUB = 9; // Username of user on github
+  USER_DATA_PRIMARY_ADDRESS_ETHEREUM = 10; // Primary address for the user on Ethereum
+  USER_DATA_PRIMARY_ADDRESS_SOLANA = 11; // Primary address for the user on Solana
```

Comment:
FYI, this is going to conflict with @aditiharini's pro changes for a banner image. One of you will need to update the sequence number